### PR TITLE
GH-569 Add home list, fix tabulation in home-side commands.

### PR DESCRIPTION
**Pull request by vLuckyyy**
Fixes: #569 

Make improvements to the code by adding a "home" list instead of displaying a "not exist" message. Additionally, fix the tab completion feature and include a notice handler for broadcasting notifications.

### CODE REVIEW

File: eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeArgument.java [hunk 1]
```diff
@@ -1,35 +1,68 @@
...
+        Option<Home> home = this.homeManager.getHome(uniqueId, argument);
+
+        if (home.isPresent()) {
+            return Result.ok(home.get());
+        }
+
+        String join = String.join(", ",
+            this.homeManager.getHomes(uniqueId).stream()
+                .map(Home::getName)
+                .toList());
```
Comment (imDMK) [inline eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeArgument.java:49]:
```suggestion
        List<String> homes = this.homeManager.getHomes(uniqueId).stream()
                .map(Home::getName)
                .toList();
                
        String join = String.join(", ", homes);
```

**Review by P1otrulla — APPROVED** [review]

**Review by Kryspek — APPROVED** [review]
fajne

File: eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeCommand.java [hunk 2]
```diff
@@ -40,9 +40,15 @@ void execute(Player player) {
         Collection<Home> playerHomes = this.homeManager.getHomes(player.getUniqueId());
 
         if (playerHomes.size() != 1) {
+            String join = String.join(", ",
+                this.homeManager.getHomes(player.getUniqueId()).stream()
+                    .map(Home::getName)
+                    .toList());
+
```
Comment (imDMK) [inline eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeCommand.java:47]:
```suggestion
            List<String> homes = this.homeManager.getHomes(player.getUniqueId()).stream()
                .map(Home::getName)
                .toList();
                
            String join = String.join(", ", homes);
```

File: eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeCommand.java [hunk 3]
```diff
@@ -59,7 +65,7 @@ void execute(Player player, @Arg Home home) {
         this.teleportToHome(player, home);
     }
 
-    private void teleportToHome(Player player, Home home) {
+    private void teleportToHome(Player player, @Arg Home home) {
```
Comment (imDMK) [inline eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeCommand.java:68]: ? 
This is not a command
```suggestion
    private void teleportToHome(Player player, Home home) {
```

Comment (Rollczi) [inline eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeCommand.java:68]: Remove this annotation

**Review by imDMK — APPROVED** [review]

**Pull request merged**